erpnext/stock/doctype/product_recall/product_recall.py

```diff
@@ -9,9 +9,15 @@
 from frappe.utils import comma_and
 
 class ProductRecall(Document):
+	def validate(self):
+		self.validate_batch_qty()
+
 	def on_submit(self):
 		self.create_product_recall_notice()
 
+	def validate_batch_qty(self):
+		if not frappe.db.get_value("Batch", self.batch_no, "batch_qty"):
```

Comment:
if batch_qty is None, then too it will throw this error, is this intentional?

Reply:
Yes if its None, then it should not proceed.